operator/model/src/main/java/io/apicurio/registry/operator/api/v3/ApicurioRegistry3Status.java

```diff
@@ -0,0 +1,54 @@
+package io.apicurio.registry.operator.api.v3;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyDescription;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import com.fasterxml.jackson.annotation.JsonSetter;
+import com.fasterxml.jackson.annotation.Nulls;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import io.apicurio.registry.operator.api.v3.status.Conditions;
+import io.apicurio.registry.operator.api.v3.status.Info;
+import io.fabric8.kubernetes.api.model.KubernetesResource;
+
+import java.util.List;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({ "conditions", "info" })
+@JsonDeserialize(using = JsonDeserializer.None.class)
+public class ApicurioRegistry3Status implements KubernetesResource {
```

Comment:
Use `io.javaoperatorsdk.operator.api.ObservedGenerationAwareStatus`. The framework then updates the observed generation automatically.

Reply:
this adds the dependency on the JOSDK to the model package ... but ok ..